src/chart/Scatter.js

```diff
@@ -0,0 +1,171 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "./XYAxis", "../api/INDChart", "css!./Scatter"], factory);
+    } else {
+        root.chart_Column = factory(root.d3, root.chart_XYAxis, root.api_INDChart);
+    }
+}(this, function (d3, XYAxis, INDChart) {
+    function Scatter(target) {
+        XYAxis.call(this);
+        INDChart.call(this);
+    }
+
+    Scatter.prototype = Object.create(XYAxis.prototype);
+    Scatter.prototype._class += " chart_Scatter";
+    Scatter.prototype.implements(INDChart.prototype);
+
+    Scatter.prototype.publish("paletteID", "default", "set", "Palette ID", Scatter.prototype._palette.switch(),{tags:['Basic','Shared']});
+    Scatter.prototype.publish("dotRepresentation", "cross", "set", "Selects the representation for the data", ["circle", "rectangle", "cross"]);
+    Scatter.prototype.publish("pointSize", 8, "number", "pointSize");
+
+    Scatter.prototype.updateChart = function (domNode, element, margin, width, height) {
+        var context = this;
+
+        this._palette = this._palette.switch(this.paletteID());
+
+        var column = this.svgData.selectAll(".data")
+            .data(this._data, function (d) { var k = d.filter(function(d, i) {return i > 0;}); return k + Math.random(); })
```

Comment:
The "key" function here is worrying, the "random" will probably force all the points to be removed and appended on _every_ update call.  Can you delete this and I can show you a better way to handle the "shape" changing issue.